src/js/tabs/osd.js

```diff
@@ -1123,7 +1131,9 @@ OSD.chooseFields = function () {
       F.MAX_ALTITUDE,
       F.BLACKBOX,
       F.BLACKBOX_LOG_NUMBER,
-      F.MAX_G_FORCE
+      F.MAX_G_FORCE,
+      F.MAX_ESC_TEMP,
```

Comment:
These can't just be added here, since they are only available for API version 1.41 and up. See above for how to add them conditionally.

Reply:
This is still not fixed.

Reply:
I seem to not understand your comment then. I thought I was meant to leave the two lines in the else statement and limit it to 1.41 like I did now. Testing this on earlier api versions these two stats show up as well, which I do understand is an issue.

Would you mind specifying?

Reply:
I thought I specified it in the conversation before, but once more:
- the branch with `if (semver.lt(CONFIG.apiVersion, "1.39.0"))` does not need any modification, because this will only be taken for firmware that is older than API 1.41;
- the `else` branch needs the two new stats to be added conditional to `semver.gte(CONFIG.apiVersion, "1.41.0")`, since these stats are only available in firmware that is API 1.41 or newer.